source/string.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+#include <aws/common/string.h>
+
+const struct aws_string * aws_string_from_c_str_new(struct aws_allocator * allocator, const char * c_str) {
```

Comment:
nit:

spacing between * and function name.

Reply:
Is this specified in a coding standard? I prefer spaces on both sides of the *. If not that, I definitely prefer omitting the space between the star and the type, since this is meant to be read (const struct aws_string *) as a return type. Omitting the space between the function name and the star would be what I prefer the least. But this is not a hill I will die on.

Reply:
It doesn't really matter to me other than consistency. It's the defacto standard at the moment, but it's not in the README at the moment.